README.md

```diff
@@ -14,8 +10,279 @@ But we need have a clear way to discover how you've been invited to a group with
 We also need to consider how to ensure our group data is replicated _enough_ - e.g. if a group is just me and my sister, then are there gonna be enough copies of the group data to be resilent/ timely.
 
 
---- 
+## Principles
+
+1. **group membership should be opaque**
+   - you shouldn't be able to guess who is in a group using public info
+     - shards people are using
+     - groupId (e.g. %a2148bs2K3klmhnaALsdaN90=.cloaked)
+2. **we support sympathetic replication**
+
+
+
+## Spec
+
+This work builds on the [ssb-meta-feeds-spec] (v1).
+
+We define two types of feeds that each peer will have:
+1. <div class='group-invite'></div> the group-invite feed
```

Comment:
I don't like the name "group invite" (the lobby feed), happy to change

Reply:
Just `invitations`?

Reply:
We could drop these `<div>` since they don't render in GitHub